lib/expression.ts

```diff
@@ -20,22 +20,83 @@ export default class ExpressionTranspiler extends base.TranspilerBase {
           this.visit(binExpr.right);
           this.emit(')');
         } else {
-          this.visit(binExpr.left);
-          if (operatorKind === ts.SyntaxKind.InstanceOfKeyword) {
-            this.emit('is');
-            this.fc.visitTypeName(<ts.Identifier>binExpr.right);
-          } else if (operatorKind == ts.SyntaxKind.InKeyword) {
-            this.reportError(node, 'in operator is unsupported');
-          } else {
-            this.emit(ts.tokenToString(binExpr.operatorToken.kind));
-            this.visit(binExpr.right);
+          var tokenStr = ts.tokenToString(operatorKind);
+          switch (operatorKind) {
```

Comment:
Needs a comment explaining this is handling bit fiddling operators, and why you need the explicit cast for Dart.